ballerina/Ballerina.toml

```diff
@@ -1,12 +1,15 @@
 [package]
 org = "ballerinax"
 name = "activemq.driver"
-version = "0.1.0"
+version = "0.1.1"
 authors = ["Ballerina"]
 keywords = ["ActiveMQ"]
 repository = "https://github.com/ballerina-platform/module-ballerinax-activemq.driver"
 license = ["Apache-2.0"]
-distribution = "2201.6.0"
+distribution = "2201.7.0"
+
+[platform.java11]
+graalvmCompatible = true
 
 [[platform.java11.dependency]]
 path = "./lib/activemq-client-5.18.2.jar"
```

Comment:
Better to add `groupId` and `artifactId` to all of this Java dependencies

Reply:
Fixed with [99ca877bc7b330e458e4689628c928e3ece748db](https://github.com/ballerina-platform/module-ballerinax-activemq.driver/pull/8/commits/99ca877bc7b330e458e4689628c928e3ece748db)